browser/ui/views/tabs/brave_tab_container.h

```diff
@@ -95,6 +97,10 @@ class BraveTabContainer : public TabContainerImpl {
 
   void UpdateLayoutOrientation();
 
+  void PaintBoundingBoxForTiles(gfx::Canvas& canvas);
+  void PaintBoundingBoxForTile(gfx::Canvas& canvas,
+                               const SplitViewBrowserData::Tile& tile);
```

Comment:
AFAIK, passing as a  pointer is chromium convention when it's updated by callee.